api/osko/v1alpha1/alertmanagerconfig_types.go

```diff
@@ -1,12 +1,14 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // AlertManagerConfigSpec defines the desired state of AlertManagerConfig
 type AlertManagerConfigSpec struct {
-	Raw map[string]string `json:"raw,omitempty"`
+	Raw       map[string]string  `json:"raw,omitempty"`
```

Comment:
we don't need raw anymore, do we? 🤔

Reply:
We don't use it at this time. It can be removed, but later something like this might become handy.

Reply:
I would remove it until we need it :shrug: